meeteval/viz/file_server.py

```diff
@@ -1,35 +1,56 @@
 """
-Deliver normalized audio files.
+A file server, that exposes wav files from the filesystem:
+ - Only delivers wav files
+ - Normalizes the audio files (Useful for far files, that is typically low volumn)
```

Comment:
Normalizes the audio files (Useful for far-field recordings that typically have a low volume)

Reply:
done